### ISSUE TITLE
ants/segmentation/label_geometry_measures.py will need updating when ANTs version is bumped

### ISSUE THREAD

**Issue by cookpa** [ANTsX/ANTsPy#612]
Physical volume of labels will be included in the output after merging

 https://github.com/ANTsX/ANTs/pull/1737
 
Currently, `label_geometry_measures.py` creates this on the fly.

**Comment by cookpa** [comment]
Understood

**Comment by cookpa** [comment]
OK used the column name "VolumeInMillimeters", Flatlanders will have to deal. 

Now the only change required will be to not duplicate the "VolumeInMillimeters" column in the function, since it will already be there.

**Comment by ntustison** [comment]
yeah, i've seen that and i'm guessing it was my fault.  would this be a problem @stnava if we fixed it?

**Comment by stnava** [comment]
here is the check if it will be a problem:

run : 
```
old=ants.label_geometry_measures( some_labels, some_intensity )
new=ants.label_geometry_measures( some_labels, some_intensity, param=new_filter )
should_be_true( names(old) %in% names(new) )
```

are they the same?     at least conceptually - that's the idea.   ie if someone expects to get a variable name - they should get it and it should be correct.

**Comment by cookpa** [comment]
Some of the variables don't exist in the supported filter, but the ones that do are named the same now

**Comment by cookpa** [comment]
Further update: I added back eccentricity and Axes length. Got everything except weighted centroid now.

**Comment by cookpa** [comment]
#782
